doc/api/fs.md

```diff
@@ -436,7 +436,7 @@ changes:
 -->
 
 * `path` {string|Buffer|URL}
-* `mode` {integer}
+* `mode` {integer} **Default:** `fs.constants.F_OK`
```

Comment:
It's good to get consistency here but I would prefer something like:

```
Defaults to `fs.constants.F_OK`
```

Without the bold styling.